go/libraries/doltcore/sqle/writer/prolly_index_writer.go

```diff
@@ -47,6 +48,7 @@ func getPrimaryProllyWriter(ctx context.Context, t *doltdb.Table, sqlSch sql.Sch
 		keyMap:  keyMap,
 		valBld:  val.NewTupleBuilder(valDesc),
 		valMap:  valMap,
+		ns:      t.NodeStore(),
```

Comment:
you can access the `NodeStore` through the `prolly.Map` rather than added a utility method to `doltdb.Table`